block-storage/src/main/scala/coop/rchain/blockstorage/BlockDagFileStorage.scala

```diff
@@ -504,14 +504,17 @@ final class BlockDagFileStorage[F[_]: Concurrent: Sync: Log: RaiseIOError] priva
                                                           Log[F].warn(
                                                             s"Block ${Base16.encode(block.blockHash.toByteArray)} sender is empty"
                                                           ) *> newValidatorsLatestMessages.pure[F]
-                                                        } else if (block.sender.size() == 32) {
+                                                        } else if (block.sender.size() == 65) {
```

Comment:
let's use a constant keyLength instead of the magic number. We can put that in the trait `SignaturesAlg`